runserver.py

```diff
@@ -2,8 +2,8 @@
 
 # -- SERVER CONFIGURATION -- (can be overridden from shell)
 
-config = (('MASTER_BRANCH', '2015'),
-          ('ALLOW_MANUAL_BUILD_TRIGGER', '0'))
+config = (('MASTER_BRANCH', '2016'),
+          ('ALLOW_MANUAL_BUILD_TRIGGER', '1'))
```

Comment:
any particular reason? People used it really often last year to rebuild their papers. Anyway, it's set to 1 in the __init__ file in procbuild. It seemed more natural for them to agree.